v6_carrier_py/wrapper.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pickle
+from io import StringIO
+
+import pandas as pd
+from SPARQLWrapper import SPARQLWrapper, CSV
+from vantage6.client import serialization
+from vantage6.tools import docker_wrapper
+from vantage6.tools.data_format import DataFormat
+from vantage6.tools.dispatch_rpc import dispact_rpc
```

Comment:
Is this really a typo in the vantage6 library?

Reply:
yeah it really is, I know :disappointed: